src/components/AddActor.jsx

```diff
@@ -31,7 +31,7 @@ const AddActor = ({ actors, setActors }) => {
   return (
     <form onSubmit={newActorHandler}>
       <h3>Add a new actor</h3>
-      <input name='actorName' placeholder="enter actor's name" />
+      <input name='actorName' maxLength="30" placeholder="enter actor's name" />
```

Comment:
we could probably add a minLength too, eh?

Reply:
Unless we want a min length greater than 1, it doesn't do much. With a minlength of 1, the name can still be input because null is also valid (not sure why it allows null, but that is what mdn says.) If we wanted a min length that was 3, then it would show a warning for 1 or 2 characters, but I don't think we want a min length greater than 1.